Straight_skeleton_2/doc/Straight_skeleton_2/CGAL/Straight_skeleton_converter_2.h

```diff
@@ -100,7 +100,7 @@ boost::shared_ptr<Target_skeleton> operator()( Source_skeleton const& s) const;
 \tparam SrcSs type of the source straight skeleton
 \tparam TgtSs type of the target straight skeleton
 \tparam NTConverter  a function object that must 
-provide `TgtSs:Traits::FT operator()(SrcSs::Traits::FT n)` that converts `n` to an 
+provide `TgtSs::Traits::FT operator()(SrcSs::Traits::FT n)` that converts `n` to an 
```

Comment:
Duplicate of #3524

Reply:
I removed it.

Reply:
You should not have removed it. When Git will try to resolve conflicts, instead of seeing two commits with the same patch, it will see two commits with contradicting patches.

Reply:
Ah yes, I'll squash it.